### ISSUE TITLE
Is the built-in integration for Lutron Caséta the long term path forward?

### ISSUE THREAD

**Issue by scpotter** [upsert/lutron-caseta-pro#80]
At this point the built-in integration for Lutron Caséta includes more features including Telnet and does not need any features from this custom component. A merge to the HA code base will not be attempted.

_Originally posted by @upsert in https://github.com/upsert/lutron-caseta-pro/issues/15#issuecomment-1030687330_

First, thank you for creating and maintaining this component. Without this I may not have ever started my HA journey. Sounds like it’s time to look into migrating to the built-in integration, both to reduce maintenance (one less thing to update) and likelihood of better enhancements and support long term. Thank you, again!

**Comment by czoog** [comment]
Agree with @scpotter .  This component is what allowed my rather large caseta install (4 pro hubs, 100 plus devices) to work.  I too will now look at migrating to the built-in integration.  Thank very much @upsert this component has proved invaluable in my HA stetup.

**Comment by cajuncoding** [comment]
Yep I agree, this component has been one of my absolute favorites and most useful (especially Pico remotes)!

**Comment by Syco54645** [comment]
I just want to say thanks to @upsert for this integration. I finally upgraded to the inbuilt version. It was quite easy. I would have never used HA if not for this integration. Thanks again!

**Comment by ccontour** [comment]
As of right now, there doesn't seem to be a super straightforward way of going from "button press on pico remote" to "message on home assistant websocket" using the built in integration. It only provides pico remotes as devices automations, which require lots of blueprints to get working, instead of a basic sensor entity + state like this integration does. 

If this integration is being deprecated, some notes on how to create something analogous to the current entities would be super helpful. For now I'm currently in a "using both integrations" camp to not lose functionality

**Comment by robertgetch** [comment]
I feel like the HA Integration still doesn't support SerenaTiltOnlyWoodBlind which is the slat version from Caseta.

**Comment by SolidOceanTrust** [comment]
Thank you @upsert for this integration!
